deploy/pipeline-nightly-update.yaml

```diff
@@ -27,6 +27,10 @@ spec:
     name: keycloak-channel
     type: string
     default: stable-v26
+  - description: Additional flags for helm install command, example '--set=kuadrant.installPlanApproval=Manual --set=kuadrant.startingCSV=kuadrant-operator.v1.2.0'
+    name: helm-install-flags
+    type: string
+    default: --set=kuadrant.installPlanApproval=Automatic
```

Comment:
If you want to have "empty" default value why not put some whitespace like " "? Bash ignores extra white spaces - I think :thinking:

Reply:
I don't like this but to be honest I don't like my solution equally :) If you like this more I can rework.